server/views/partials/overviewPage/courtCases/courtCasesAndReleaseDates.njk

```diff
@@ -40,26 +40,31 @@
 
     <div data-qa="overview-latest-calculation" class="govuk-grid-column-full">
       <h3 class="card-subheading govuk-!-margin-top-2 govuk-!-margin-bottom-1">Latest calculation</h3>
-      {% if courtCaseSummary.latestCalculation %}
-        {{ govukSummaryList({
-          classes: 'govuk-summary-list--no-border',
-          rows: [
-            {
-              key: { text: "Date of calculation" },
-              value: { text: courtCaseSummary.latestCalculation.calculationDate | formatDate('long') }
-            },
-            {
-              key: { text: "Establishment" },
-              value: { text: courtCaseSummary.latestCalculation.establishment or "Not entered"  }
-            },
-            {
-              key: { text: "Calculation reason" },
-              value: { text: courtCaseSummary.latestCalculation.reason }
-            }
-          ]
-        }) }}
+      {% if courtCaseSummary.latestCalculation.status === 'fulfilled' %}
```

Comment:
Now working directly with a PromiseSettledResult.  I was thinking maybe of making this terser by using a filter to check the error condition